### PR TITLE
fix(learn): removed space from first sentence and added instruction to basic javascript challenges

### LINKED ISSUE #42455: Translation: backticked strings loose spaces at the end
I think we have a small issue with translations and spaces at the end of strings in backticks
In at least one challenge there is a space at the end of a string, (the string `This is the start. `)
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md
in the english challenge it is kept
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator

It doesn't appear in the string to translate on crowdin tho

image

in consequence the spanish and chinese translation don't have that space

https://www.freecodecamp.org/espanol/learn/javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator
https://chinese.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator

The next challenge has the same issue.
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-the-plus-equals-operator

### PR DESCRIPTION
**Pull request by patlehmann1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #42455

### CODE REVIEW

**Comment by patlehmann1** [comment]
Yeah, I'm looking in gitpod now and the suggestion referenced in #42455 doesn't seem to resolve the issue. I'll try to think of a better solution, please let me know if you have anything in mind.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
I don't know if the issue is fixed or not on Crowdin side, as there we would need to have the file fed to the Crowdin pipeline, but I don't think this is the best option.

The code snippet looks totally out of place now
image

production is like this
image

**Comment by patlehmann1** [comment]
Yeah, that's the behavior I was seeing also. Not too familiar with Crowdin.

**Comment by ieahleen** [comment]
Do not worry about Crowdin, that's not something you can test. If you can find a way using `pre` that would not create a code block, that would be ideal and we can try it out.

**Comment by patlehmann1** [comment]
I'll see what I can do.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ var ourStr = "I come first. " + "I come second.";
 The string `I come first. I come second.` would be displayed in the console.
 # --instructions--
 
-Build `myStr` from the strings `This is the start. ` and `This is the end.` using the `+` operator.
+Build `myStr` from the strings <pre>`This is the start. `</pre> and `This is the end.` using the `+` operator.
```
Comment (ieahleen) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: This still doesn't work, it appears as before

Maybe it's actually one of the crowdin workflows that needs to be used for this 🤷

Comment (patlehmann1) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: Yeah, I'm stumped myself. Please let me know if you have any ideas. I can try to jump into the crowdin docs and get familiar with it, if needed.

Comment (ieahleen) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: @nhcarrigan is the Crowdin expert, maybe he have some ideas

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: Honestly might be better to reword the challenge a bit.

Comment (ieahleen) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: maybe adding quotes?
I know we were removing those from strings but maybe here it's best for translating

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md:31]: Or:
> "Build `myStr` from the strings `This is the start.` and `This is the end.` using the `+` operator. Be sure to include a space between the two strings.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-the-plus-equals-operator.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ ourStr += "I come second.";
 
 # --instructions--
 
-Build `myStr` over several lines by concatenating these two strings: `This is the first sentence. ` and `This is the second sentence.` using the `+=` operator. Use the `+=` operator similar to how it is shown in the example. Start by assigning the first string to `myStr`, then add on the second string.
+Build `myStr` over several lines by concatenating these two strings: <pre>`This is the first sentence. `</pre> and `This is the second sentence.` using the `+=` operator. Use the `+=` operator similar to how it is shown in the example. Start by assigning the first string to `myStr`, then add on the second string.
```
Comment (ieahleen) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-the-plus-equals-operator.md:27]: @nhcarrigan would you say something similar here too? adding the "Be sure to add a space between the two strings" hre too?

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-the-plus-equals-operator.md:27]: Yep.

**Comment by ieahleen** [comment]
@patlehmann1 are you still up to it? the direction of the changes has changed a bit

**Comment by patlehmann1** [comment]
Ok, so just to be clear here. I am removing the `pre` formatting from the challenge and adding that additional sentence to the end? "Be sure to include a space between the two strings."?

**Comment by ieahleen** [comment]
@patlehmann1 and removing the space from the first string too. Make sure it says that you have to add the space between the strings

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍 thank you

Though I would change the PR title 😉

**Comment by patlehmann1** [comment]
Good catch @ieahleen! Will do.

**Comment by ieahleen** [comment]
You can from here:
image

**Comment by patlehmann1** [comment]
Done

**Review by naomi-lgbt — APPROVED** [review]
This looks good to me.

**Pull request merged**